### PR TITLE
Add TooManyRequests error

### PR DESCRIPTION
**Pull request by m3nd3s**
This PR adds a new error `Too Many Requests` when  API returns 429 error. 

See [API request limit](https://developers.rdstation.com/en/request-limit) for more details

### CODE REVIEW

**Review by Seralto — APPROVED** [review]
LGTM

**Comment by Seralto** [comment]
Needs to bump version in `lib/rdstation/version.rb`

File: README.md [hunk 1]
```diff
@@ -315,6 +315,7 @@ Each endpoint may raise errors accoording to the HTTP response code from RDStati
 - `RDStation::Error::Conflict` (409)
 - `RDStation::Error::UnsupportedMediaType` (415)
 - `RDStation::Error::UnprocessableEntity` (422)
+- `RDStation::Error::ToManyRequests` (429)
```
Comment (jeveaux) [inline README.md:318]: Ficou um typo aqui :) `ToMany` - `TooMany`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 2.4.0
+
+- Add the ToManyRequests errors in case of rate limit exceeded. See [API request limit](https://developers.rdstation.com/en/request-limit) for more details
```
Comment (jeveaux) [inline CHANGELOG.md:3]: Ficou um typo aqui :) `ToMany` - `TooMany`

Comment (m3nd3s) [reply CHANGELOG.md:3]: Opa, vlw!

Comment (m3nd3s) [reply CHANGELOG.md]: Fixed

File: README.md (hunk 1 above)
Comment (m3nd3s) [reply README.md]: Agora sim, tudo corrigido

**Review by jeveaux — APPROVED** [review]
:shipit:

**Pull request merged**